skin-resque-frontend/src/pages/FrontPage/FrontPage.js

```diff
@@ -1,8 +1,55 @@
 import './FrontPage.scss';
 import photo from '../../resources/front-page-skin-photo.jpg';
-import Button from '../../components/Button/Button'
+import Button from '../../components/Button/Button';
+import Text from '../../components/Text/Text';
+import Heading from '../../components/Heading/Heading';
+import Arrow from '../../components/Arrow/Arrow';
+import { useState } from 'react';
 
 function FrontPage() {
+
+  const ingredients = [
+    {
+      "id": 1,
+      "name": "rosemary",
+      "description": `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+      ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
+      in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, 
+      sunt in culpa qui officia deserunt mollit anim id est laborum.`,
+      "photo": '/images/rosemary.jpg',
+    },
+    {
+      "id": 2,
+      "name": "rosemary2",
+      "description": `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+      ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
+      in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, 
+      sunt in culpa qui officia deserunt mollit anim id est laborum.`,
+      "photo": '/images/rosemary.jpg',
+    }
+  ]
+
+  const [showIngredient, setShowIngredient] = useState(ingredients[0])
```

Comment:
Wouldn't simple index state be easier and quicker to do?

Reply:
I am not sure about this